randovania/games/dread/json_data/Dairon.txt

```diff
@@ -1670,10 +1536,10 @@ Extra - asset_id: collision_camera_018
   * Extra - right_shield_def: None
   > Dock to Big Hub
       Morph Ball and After Dairon - Central Unit
-  > Tile Group (SPEEDBOOST)
```

Comment:
Remove this connection: "Dock to EMMI Zone Exit North" -> "Lower-Left Area"
Add this connection: "Dock to EMMI Zone Exit North" -> "Door to EMMI Zone Exit East", with a speedbooster requirement

Reply:
I did this because speed can get you down there, while you are not guaranteed to have charge (I have been in this situation). If you were to stop your speed before going through the flash gate, you'd have to go back and get the speed again. And then it's trivial to go from Lower-left to the charge door, so it's essentially the same. But i can still add the connection from Exit North to Exit East if you feel it's necessary.